stages/org.osbuild.ostree.deploy

```diff
@@ -61,6 +61,15 @@ SCHEMA_2 = """
       "description": "optional OStree remote to use for the deployment",
       "type": "string"
     },
+    "target_signature_verification": {
+      "description": "Set signature verification for ostree commit and container image updates",
+      "type": "string",
+      "enum": [
+        "ostree-remote",
+        "container-signing",
+        "none"
+      ]
+    },
```

Comment:
After reading the whole discussion and thinking about this more, I'm wondering if we can go with a more "raw" solution here and make the verification type part of the `remote`.

Here's my thinking:
- `target_signature_verification` requires that `remote` is specified.
- The values of `target_signature_verification`, `remote` (sometimes), and `image_name` are concatenated to become the value to `target-imgref`.
- The option is unused when the input is a commit (and we don't produce an error when it's defined and unused).

If we treat `remote` as an alias for `target-imgref` (in the container case), we can pass it through and leave it up to the manifest generator to define all the bits.  So the following would be valid values for the `"remote"` (when the input is a container):
- `ostree-remote-registry:fedora:quay.io/fedora/fedora-coreos:stable`
- `ostree-image-signed:registry:quay.io/fedora/fedora-coreos:stable`
- `ostree-unverified-registry:quay.io/fedora/fedora-coreos:stable`

(and the other variants/synonyms).

(We could also support implicit values for the verification and fall back to the default if it's not specified, but that might be too "smart" and flaky since the number of components separated by `:` isn't always the same)

This also enables an extra bit of flexibility with the remote setup.  A manifest can be generated that pulls an ostree container from one source, for example `skopeo.items.<hash>.image.name = "https://koji.fedoraproject.org/.../fedora/fedora-coreos:stable"`, and sets up the remote to track a different source, for example `stage.options.remote = "ostree-image-signed:registry:quay.io/fedora/fedora-coreos:stable"`, so we could build images by deploying a container directly from the build system but have them set up to track a public container registry instead.

Reply:
We have added a separate stage now.